src/util/FileMonitor.h

```diff
@@ -37,38 +36,36 @@ namespace lsst::qserv::util {
 /// events wait for.
 class FileMonitor {
 public:
-	using Ptr = std::shared_ptr<FileMonitor>;
+    using Ptr = std::shared_ptr<FileMonitor>;
```

Comment:
This declaration is useless since the rest of the code is using:
```
std::shared_ptr<FileMonitor>
```
We have this historical artifact in many places of the Qserv code. We may need to get rid of it (eventually).

Reply:
I disagree. I find it is convenient shorthand, but in many cases it can't be used because the header file has not been included, usually in other header files.

Reply:
I may leave this definition. My point was that it's not used. No reason to have elements of an interface that aren't used. These elements complicate the interface w/o providing any benefits to its users.